discourse-setup

```diff
@@ -576,7 +621,7 @@ ask_user_for_config() {
     sed -i -e "s/$src/$dst/w $changelog" $web_file
     if [ -s $changelog ]
     then
-  echo "web.ssl.template.yml enabled"
+      echo "web.ssl.template.yml enabled"
```

Comment:
indentation problem?

Reply:
There is some other odd indentation. Some editor that I use, for example, is indenting 4 spaces after a `then` but only 2 spaces after an `else`. There are other examples of 2 vs 4 space indents. 

Perhaps I (or someone else?) should do another fix-the-formatting PR after we've worked out functional changes in this commit. I'm not clear if there are prescribed standards for formatting `bash` files.